docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,248 @@
 title: Quickstart to run Loki locally
 menuTitle: Loki quickstart
 weight: 200
-description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+description: How to create and use a local Loki cluster for testing and evaluation purposes.
+killercoda:
+  title: Loki Quickstart Demo
+  description: This sandbox provides an online enviroment for testing the Loki quickstart demo.
+  details:
+    finish:
+      text: finished.md
+  backend:
+    imageid: ubuntu
 ---
 
+<!-- Killercoda intro.md START -->
+
 # Quickstart to run Loki locally
 
 If you want to experiment with Loki, you can run Loki locally using the Docker Compose file that ships with Loki. It runs Loki in a [monolithic deployment](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode) mode and includes a sample application to generate logs.
 
 The Docker Compose configuration instantiates the following components, each in its own container:
 
-- **flog** a sample application which generates log lines.  [flog](https://github.com/mingrammer/flog) is a log generator for common log formats.
+- **flog** a sample application which generates log lines.
+  [flog](https://github.com/mingrammer/flog) is a log generator for common log formats.
 - **Grafana Alloy** which scrapes the log lines from flog, and pushes them to Loki through the gateway.
-- **Gateway** (NGINX) which receives requests and redirects them to the appropriate container based on the request's URL.
+- **Gateway** (nginx) which receives requests and redirects them to the appropriate container based on the request's URL.
 - One Loki **read** component (Query Frontend, Querier).
 - One Loki **write** component (Distributor, Ingester).
 - One Loki **backend** component (Index Gateway, Compactor, Ruler, Bloom Compactor (Experimental), Bloom Gateway (Experimental)).
 - **Minio** an S3-compatible object store which Loki uses to store its index and chunks.
 - **Grafana** which provides visualization of the log lines captured within Loki.
 
-{{< figure max-width="75%" src="/media/docs/loki/get-started-flog-v3.png" caption="Getting started sample application" alt="Getting started sample application">}}
+{{< figure max-width="75%" src="/media/docs/loki/get-started-flog-v3.png" caption="Getting started sample application" alt="Getting started sample application" >}}
+
+<!-- Killercoda intro.md END -->
```

Comment:
I see we added intro.md as a section here. Can we also do this with the finish.md or does it need to be in details ?

Reply:
We can do it with either, though if it's specified in the front matter, it's assumed to exist in the killercoda repository and the tool doesn't bother looking for the `finish.md` contents within this page.